config/telemetry/default.yml

```diff
@@ -0,0 +1,37 @@
+telemetry:
+  - id: MySQLVersion
```

Comment:
i thought that we sign each telemetry field, aren't we?

Reply:
no, as of now we agreed to sign the whole file.